src/parsers/power_models_data.jl

```diff
@@ -900,41 +902,146 @@ function read_3w_transformer!(
     end
 end
 
-function make_dcline(name::String, d::Dict, bus_f::ACBus, bus_t::ACBus)
-    return TwoTerminalHVDCLine(;
+function make_dcline(name::String, d::Dict, bus_f::ACBus, bus_t::ACBus, source_type::String)
+    if source_type == "pti"
+        return TwoTerminalLCCLine(;
+            name = name,
+            available = d["available"],
+            arc = Arc(bus_f, bus_t),
+            active_power_flow = get(d, "pf", 0.0),
+            r = d["r"],
+            transfer_setpoint = d["transfer_setpoint"],
+            scheduled_dc_voltage = d["scheduled_dc_voltage"],
```

Comment:
Could do some `kwargs...` splatting to streamline this arguments list

Reply:
... maybe next time?